core/include/detray/utils/ranges/ranges.hpp

```diff
@@ -0,0 +1,313 @@
+/** Detray library, part of the ACTS project (R&D line)
+ *
+ * (c) 2022 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+#pragma once
+
+// Thrust include(s)
+#if defined(__CUDACC__)
```

Comment:
Can we remove this macro? Or the thrust headers cannot be included without CUDA environment?

Reply:
No. I think I cannot come up with a better way to do this right now :p

Reply:
This does not work at the moment anyways, so I commented it out for now :(